ui/src/app/pages/profile/data-use-agreement-panel.tsx

```diff
@@ -0,0 +1,139 @@
+import {Button} from 'app/components/buttons';
+import {FlexColumn} from 'app/components/flex';
+import {TextInput} from 'app/components/inputs';
+import {TooltipTrigger} from 'app/components/popups';
+import {SpinnerOverlay} from 'app/components/spinners';
+import colors, {colorWithWhiteness} from 'app/styles/colors';
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+import {Modal, ModalBody, ModalFooter, ModalTitle} from '../../components/modals';
+import {dataUseAgreementStyles, IndentedListItem, IndentedUnorderedList, SecondHeader} from './data-use-agreement';
+
+const DuaTextInput = (props) => {
+  // `fp.omit` used to prevent propagation of test IDs to the rendered child component.
+  return <TextInput {...fp.omit(['data-test-id'], props)}
+                    style={{
+                      padding: '0 1ex',
+                      width: '12rem',
+                      fontSize: 10,
+                      borderRadius: 6,
+                      ...props.style
+                    }}/>;
+};
+
+const InitialsAgreement = (props) => {
+  return <div style={{display: 'flex', marginTop: '0.5rem'}}>
+    <DuaTextInput onChange={props.onChange} value={props.value}
+                  placeholder='INITIALS' data-test-id='dua-initials-input'
+                  style={{width: '4ex', textAlign: 'center', padding: 0}}/>
+    <div style={{marginLeft: '0.5rem'}}>{props.children}</div>
+  </div>;
+};
+
+export const SanctionModal = (props) => {
+  return <Modal width={750}>
+    <ModalTitle style={dataUseAgreementStyles.sanctionModalTitle}>
+      All of Us Research Program - Sanctions on Violations of the Code of Conduct
+    </ModalTitle>
+    <ModalBody>
+      <label style={dataUseAgreementStyles.modalLabel}>
+        The Resource Access Board (RAB) of the <i>All of Us</i> Research Program determines whether
+        an investigator has violated the Code of Conduct outlined in the Data Use Agreement
+        signed by each user. The RAB notifies the All of Us Research Program office of the
+        violation.
+        <IndentedUnorderedList>
+          <li>
+            <label>The <i>All of Us</i> Research Program office and/or the All of Us IRB may
+              implement the following sanctions if it is determined that an investigator has violated
+              the Code of Conduct:
+            </label>
+            <IndentedUnorderedList>
+              <IndentedListItem>
+                Determine whether any action by the investigator is required to remedy the
+                violation.
+              </IndentedListItem>
+              <IndentedListItem>
+                Revoke and/or deny access of the violator to all non-public (Registered and
+                Controlled tier) <i>All of Us</i> data.
+              </IndentedListItem>
+              <IndentedListItem>
+                Post the name and affiliation of the violator on a public <i>All of Us</i>
```

Comment:
There's a space issue here (same as before, with the placement of the newline affecting whitespace trimming)